meteor.bat

```diff
@@ -0,0 +1,21 @@
+@echo off
+
+rem only if we are running from a checkout
+IF EXIST "%~dp0\.git" (
+  rem if dev_bundle is not present, get it
+  IF NOT EXIST "%~dp0\dev_bundle" (
+    PowerShell.exe -version 2.0 -file "%~dp0\scripts\windows\download-dev-bundle.ps1"
```

Comment:
error handling here sucks. It might be OK as it only runs in devel mode but we might need to check for `%errorlevel%` before proceeding or adding something like `|| exit 1` to the end.